tritond/main_test.go

```diff
@@ -0,0 +1,86 @@
+package tritond
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/pebbe/zmq4"
+	"github.com/tinylib/msgp/msgp"
+)
+
+var gConsumer *consumer
```

Comment:
what is this? Oh, I see, it is an instance of the consumer defined below. A bit hard to read.